tests/e2e/README.md

```diff
@@ -143,7 +143,7 @@ Please refer to `tests/e2e/initialization/README.md`
 ### To build the debug Osmosis image
 
 ```sh
-    make docker-build-e2e-debug
+    make test-e2e-debug
```

Comment:
The one you linked runs the test but this seems to be referring to building the image

```suggestion
    make docker-build-debug
```

Reply:
ah fixed the wrong line in code thanks for catching this!